godot-macros/src/gdextension.rs

```diff
@@ -54,6 +54,11 @@ pub fn attribute_gdextension(item: venial::Item) -> ParseResult<TokenStream> {
             let script = std::ffi::CString::new(concat!(
                 "var pkgName = '", env!("CARGO_PKG_NAME"), "';", r#"
                 var libName = pkgName.replaceAll('-', '_') + '.wasm';
+                if (!(libName in LDSO.loadedLibsByName)) {
+                    // Always print to console, avoid problems with automatic error suppression.
+                    console.error(`gdext could not find the Wasm module '${libName}', needed to load the '${pkgName}' crate's GDExtension. Please ensure a file named '${libName}' exists in the game's web export files.`);
```

Comment:
> needed to load the '${pkgName}' crate's GDExtension.

Does the "GDExtension" part really add something, or could this be simplified to

> needed to load the '${pkgName}' crate.

Reply:
Also maybe you can call it "godot-rust" instead of "gdext". This could be useful in case someone uses multiple languages or extensions.

Reply:
I agree! It looks like this now:

![image](https://github.com/user-attachments/assets/cc6efc0f-d7d4-401d-9e91-65caaa9139db)